plugins/plugin-site/src/components/PluginHealthScore.jsx

```diff
@@ -1,63 +1,101 @@
-import React, {useState} from 'react';
+import React from 'react';
 import PropTypes from 'prop-types';
-import {Progress, Tooltip} from 'reactstrap';
 
-function PluginHealthScore({healthScore, name}) {
-    const score = healthScore.value || 0;
-    const color =
-    score > 80 ? 'success' : score > 60 ? 'warning' : 'danger';
+import './PluginHealthScore.css';
 
-    const tooltipId = `tooltip-${name}`;
-    const [tooltipOpen, setTooltipOpen] = useState(false);
-    const toggle = () => setTooltipOpen(!tooltipOpen);
+function ScoreValue({value}) {
     return (
-        <>
+        <div>
+            {`${value}/100`}
```

Comment:
Maybe this could use % instead of /100? Unless you plan to have different max values for each score...

Reply:
I don't plan to have other max value, or this would be revisited by also providing the max.

Reply:
fixed in https://github.com/jenkins-infra/plugin-site/pull/1431/commits/12148caa6fc45471deb5c88d198c6d89259d5d09